packages/data/json/buildingCode.json

```diff
@@ -1,150 +1,194 @@
+                        ]
+                      }
+                    ]
+                  },
+                  {
+                    "numberReference": "9.9.9.3.(2)",
+                    "description": "Where a <defined-term>dwelling unit</defined-term> is located above another <defined-term>dwelling unit</defined-term> or common space in a house with a <defined-term>secondary suite</defined-term>, the upper <defined-term>dwelling unit</defined-term> shall be provided with a second  and separate <defined-term>means of egress</defined-term> where an egress door from that <defined-term>dwelling unit</defined-term> opens onto an exterior passageway that",
+                    "clauses": [
+                      {
+                        "numberReference": "9.9.9.3.(2)(a)",
+                        "description": "has a floor assembly with a <defined-term>fire-resistance rating</defined-term> less than 45 min,"
+                      },
+                      {
+                        "numberReference": "9.9.9.3.(2)(b)",
+                        "description": "is served by a single <defined-term>exit</defined-term> stairway or <defined-term>ramp</defined-term>, and"
+                      },
+                      {
+                        "numberReference": "9.9.9.3.(2)(c)",
+                        "description": "is located more than 1.5 m above adjacent ground level."
+                      }
+                    ]
+                  },
+                  {
+                    "numberReference": "9.9.9.3.(3)",
+                    "description": "For <defined-term override-term='dwelling unit'>dwelling units</defined-term> in a house with a <defined-term>secondary suite</defined-term> where an egress door from either <defined-term>dwelling unit</defined-term> opens onto a shared egress facility served by a single <defined-term>exit</defined-term> stairway or <defined-term>ramp</defined-term>, other than as described in <building-code-modal reference='9.9.9.3.(2)'>Sentence (2)</building-code-modal>, a second and separate <defined-term>means of egress</defined-term> need not be provided if the <defined-term>building</defined-term> is <defined-term>sprinklered</defined-term> or if the <defined-term override-term='dwelling unit'>dwelling units</defined-term> have separate and direct access from each <defined-term>storey</defined-term> to",
+                    "clauses": [
+                      {
+                        "numberReference": "9.9.9.3.(3)(a)",
+                        "description": "a balcony, or"
+                      },
+                      {
+                        "numberReference": "9.9.9.3.(3)(b)",
+                        "description": " an openable window conforming to sentences <building-code-modal reference='9.9.9.1.(2)(a)'>9.9.9.1.(2)(a)</building-code-modal> and <building-code-modal reference='9.9.9.1.(2)(b)'>(b)</building-code-modal>."
```

Comment:
I believe in the building code this says "Clauses" instead of "sentences" in this spot. (I'm not quire sure why though..)